config/Upgrade/services.yaml

```diff
@@ -81,6 +84,9 @@ services:
                 param12: '@create_pr.step',
                 param13: '@checkout.step'
              }
+            - {
+                param1: '@feature_package.fixer',
```

Comment:
We don't need it in the `composer strategy` 
the strategy just runs `composer update and couldn't be conflicts during the command

Reply:
If we don't have this issue here we won't run fixer.